arch/arm/soc/st_stm32/stm32l0/Kconfig.defconfig.series

```diff
@@ -0,0 +1,23 @@
+# Kconfig - ST Microelectronics STM32L0 MCU line
+#
+# Copyright (c) 2018 Endre Karlson <endre.karlson@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_SERIES_STM32L0X
+
+source "arch/arm/soc/st_stm32/stm32l0/Kconfig.defconfig.stm32l0*"
+
+config SOC_SERIES
+	default stm32l0
+
+if GPIO_STM32
```

Comment:
Why ports D and E are missing?

Reply:
Because I was told that they where provided by the stm32 config?

Reply:
This is true for A B C only.
E and D are needed there indeed

Reply:
Are there ports D and E in all L072s? Or that doesn't matter?

Reply:
Sorry I've been too fast, you're right.
Some L0xx support only ABC, so DEH should go in Kconfig.defconfig.stm32l072xz.